src/utils.py

```diff
@@ -268,6 +268,76 @@ def wrap_atom_predicates(atoms: Collection[LiftedOrGroundAtom],
     return new_atoms
 
 
+class LinearChainParameterizedOption(ParameterizedOption):
+    """A parameterized option implemented via a sequence of "child"
+    parameterized options.
+
+    This class is meant to help ParameterizedOption manual design.
+
+    The children are executed in order starting with the first in the sequence
+    and transitioning when the terminal function of the respect child is hit.
+
+    The children are assumed to chain together, so the initiable of the next
+    child should always be True when the previous child terminates. If this
+    is not the case, an AssertionError is raised.
+
+    The children must all have the same types and params_space, which in turn
+    become the types and params_space for this ParameterizedOption.
+
+    The options have memory, which stores the current child index.
```

Comment:
What do you mean by "The options"? Do you mean "A LinearChainParameterizedOption has memory"?